Makefile

```diff
@@ -183,10 +183,12 @@ ci-temporal-integ-test:
 	$Q go test -v ./integ  -cover -coverprofile coverage.out -coverpkg ./service/... -search=false -cadence=false -dependencyWaitSeconds=60
 
 ci-all-tests:
-	$Q go test -v ./... -cover -coverprofile coverage.out -coverpkg ./service/...
+ 	# Fails CI when used with -coverprofile flag due to tests that panic; see https://go.dev/doc/build-cover#panicprof
+ 	# $Q go test -v ./... -timeout 15m -cover -coverprofile coverage.out -coverpkg ./service/...
+	$Q go test -v ./... -timeout 15m
```

Comment:
@longquanzheng CI issue is caused by Go upgrade from 1.19 to 1.22

> Beginning in Go 1.20, Go supports collection of coverage profiles from applications and from integration tests, larger and more complex tests for Go programs.



> If my program panics, will coverage data be written?[¶](https://go.dev/doc/build-cover#panicprof)
> Programs built with go build -cover will only write out complete profile data at the end of execution if the program invokes os.Exit() or returns normally from main.main. If a program terminates in an unrecovered panic, or if the program hits a fatal exception (such as a segmentation violation, divide by zero, etc), profile data from statements executed during the run will be lost.

When tests panic, the task returns "Error 1" which fails the Github Action. We should fix it by refactoring tests, but I removed the coverage options to unblock now for

https://go.dev/doc/build-cover#panicprof

Reply:
https://github.com/indeedeng/iwf/issues/446